### ISSUE TITLE
Add RoboVM-LicenseManager screenshot

### ISSUE THREAD

**Issue by henricm** [robovm/robovm-docs#93]
Add screenshot from menu selection in license activation docs for Elcipse and IntelliJ

http://docs.robovm.com/getting-started/eclipse.html#activating-your-license
etc

**Comment by ajhager** [comment]
I assume you mean "License Manager" being selected from the "RoboVM" menu?

**Comment by henricm** [comment]
Yes, I mean a step before the one that is there now showing where to find the License Manager in the Eclipse (or IntelliJ) menu.

![skarmavbild 2015-09-18 kl 16 49 56](https://cloud.githubusercontent.com/assets/3715975/9962314/53c6a404-5e25-11e5-8915-f806196b13d9.png)
